lib/datasource/docker/index.ts

```diff
@@ -66,9 +66,14 @@ export function getRegistryRepository(
   registryUrl: string
 ): RegistryRepository {
   if (registryUrl !== defaultRegistryUrls[0]) {
-    const registry = registryUrl.replace('https://', '');
-    const registryEndingWithSlash = registry.replace(/\/?$/, '/');
+    const registryEndingWithSlash = registryUrl
+      .replace(/^https?:\/\//, '')
+      .replace(/\/?$/, '/');
```

Comment:
```suggestion
    const registryEndingWithSlash = ensureTrailingSlash(registryUrl.replace(/^https?:\/\//, ''));
```

It's in `util/url`. maybe needs running prettier.

Reply:
Thanks for the tip - I've made that change. :+1: